api-report/map.api.md

```diff
@@ -184,7 +184,7 @@ export interface IValueChanged {
 // @public
 export class LocalValueMaker {
```

Comment:
The local  value stuff really shouldn't be exported at all - can we just remove it from the exports?

Reply:
Happy to. But yeah, will need to happen in next. As soon as this PR works its way through the main->next integration, I'll get started on a big breaking change 😋